manifests/operator-service-account-rbac.yaml

```diff
@@ -37,6 +37,9 @@ rules:
   - create
   - delete
   - get
+  - list
+  - patch
+  - watch # for Spilo replica
```

Comment:
I don't recall cases of watching for endpoints within the operator

Reply:
I doubled-checked, omitting this line raises the following exception in a _replica_:

> Traceback (most recent call last):                                                                  [5/1916]
  File "/usr/local/lib/python3.6/dist-packages/patroni/dcs/kubernetes.py", line 396, in watch
    _request_timeout=(1, timeout + 1)):
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/watch/watch.py", line 115, in stream
    resp = func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/patroni/dcs/kubernetes.py", line 50, in wrapper
    return getattr(self._api, func)(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/apis/core_v1_api.py", line 12528, in list_n
amespaced_endpoints
    (data) = self.list_namespaced_endpoints_with_http_info(namespace, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/apis/core_v1_api.py", line 12630, in list_n
amespaced_endpoints_with_http_info
    collection_formats=collection_formats)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/api_client.py", line 335, in call_api
    _preload_content, _request_timeout)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/api_client.py", line 148, in __call_api
    _request_timeout=_request_timeout)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/api_client.py", line 371, in request
    headers=headers)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/rest.py", line 250, in GET
    query_params=query_params)
  File "/usr/local/lib/python3.6/dist-packages/kubernetes/client/rest.py", line 240, in request
    raise ApiException(http_resp=r)
kubernetes.client.rest.ApiException: (403)
Reason: Forbidden
HTTP response headers: HTTPHeaderDict({'Content-Type': 'application/json', 'X-Content-Type-Options': 'nosnif
f', 'Date': 'Thu, 24 May 2018 15:20:53 GMT', 'Content-Length': '285'})
HTTP response body: b'{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"endpoin
ts is forbidden: User \\"system:serviceaccount:default:zalando-postgres-operator\\" cannot watch endpoints i
n the namespace \\"default\\"","reason":"Forbidden","details":{"kind":"endpoints"},"code":403}\n'